js/acquia_lift.v2.js

```diff
@@ -7,7 +7,7 @@
 
 (function ($, Drupal) {
   var instance;
-  Drupal.acquiaLiftV2API = (function () {
+  Drupal.acquiaLiftAPI = (function () {
```

Comment:
Oh - yes! Definitely an oversight. There should be no concept of "v2" anywhere anymore. Was getting rid of it piece by piece so obviously some last bits still hadn't been dealt with...